build/shade/_k-standard-goals.shade

```diff
@@ -32,6 +32,18 @@ default SAMPLES_PROJECT_GLOB = "samples/*/*.csproj"
   }
 }
 
+functions
+  @{
+    // convenince method as we incrementally move to MSBuild standard lifecycle
+    void MSBuild(string arguments)
+    {
+      Dotnet("msbuild /nologo"
+             + " \"" + Path.Combine(Directory.GetCurrentDirectory(), ".build/targets/makefile.proj") + "\""
+             + " \"/p:LifecycleType=Standard;Configuration=" + E("Configuration") + "\""
```

Comment:
With changes to `build/targets/makefile.proj`, does this command-line lifecycle choice still work?